generators/client/templates/vue/src/test/javascript/spec/app/core/ribbon/ribbon.component.spec.ts.ejs

```diff
@@ -46,25 +42,9 @@ describe('Ribbon', () => {
     expect(ribbon.ribbonEnv).toBeNull();
   });
 
-  it('should not have ribbonEnv when no activeProfiles', async () => {
-    await wrap({ data: {} });
-    expect(ribbon.ribbonEnv).toBeNull();
-  });
-
-  it('should not have ribbonEnv when empty activeProfiles', async () => {
```

Comment:
Please move the deleted tests to principal test

Reply:
writing the test, as it does not exists !

Reply:
Please don't, principal will be refactor for oauth2 support, I'll do the test for this one. And yes, principal is indeed currently missing.

Reply:
ok